<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>modelo_182</class>
 <widget class="QDialog" name="modelo_182">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>962</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modelo 182</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>NIF:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="niflineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>EJERCICIO:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="ejercicio_comboBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLineEdit" name="nombrelineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Persona con quien relacionarse</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Teléfono:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="telefonolineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_17">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Apellidos y nombre</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="nombrerellineEdit">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="compl_sus_groupBox">
       <property name="title">
        <string>Complementaria / Sustitutiva</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QRadioButton" name="sustitutiva_radioButton">
          <property name="text">
           <string>Sustituti&amp;va</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="complementaria_radioButton">
          <property name="text">
           <string>Co&amp;mplementaria</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Núm.declaración anterior:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="anteriorlineEdit"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QComboBox" name="naturaleza_comboBox">
     <item>
      <property name="text">
       <string>1. Entidad Beneficiaria incentivos Título III Ley 49/2002 de r.f. de entidades sin fines lucrativos e incentivos mecenazgo</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2. Fundación o asociación declarada de utilidad pública art. 68.3.b) Ley IRPF</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3. Titular o administrador de un patrimonio protegido Ley 41/2003 de protección patrimonial de personas con discapacidad</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4. Partidos Políticos, Federaciones, Coaliciones o Agrupaciones de Electores</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_2">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="genera_pushButton">
       <property name="text">
        <string>&amp;Genera 182</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/aeat.png</normaloff>:/iconos/graf/aeat.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="carga_pushButton">
       <property name="text">
        <string>&amp;Cargar datos</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/refresh.xpm</normaloff>:/iconos/graf/refresh.xpm</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+C</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="borrarpushButton">
       <property name="text">
        <string>&amp;Borrar línea</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/borrar.png</normaloff>:/iconos/graf/borrar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="datospushButton">
       <property name="text">
        <string>&amp;Datos cuenta</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/text-x-vcard.png</normaloff>:/iconos/graf/text-x-vcard.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mayorpushButton">
       <property name="text">
        <string>&amp;Mayor</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/mayorg.xpm</normaloff>:/iconos/graf/mayorg.xpm</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copiarpushButton">
       <property name="text">
        <string>C&amp;opiar</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/copiar.png</normaloff>:/iconos/graf/copiar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="salirpushButton">
       <property name="text">
        <string>&amp;Salir</string>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/application-exit.png</normaloff>:/iconos/graf/application-exit.png</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+S</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>niflineEdit</tabstop>
  <tabstop>ejercicio_comboBox</tabstop>
  <tabstop>nombrelineEdit</tabstop>
  <tabstop>telefonolineEdit</tabstop>
  <tabstop>nombrerellineEdit</tabstop>
  <tabstop>compl_sus_groupBox</tabstop>
  <tabstop>sustitutiva_radioButton</tabstop>
  <tabstop>complementaria_radioButton</tabstop>
  <tabstop>anteriorlineEdit</tabstop>
  <tabstop>naturaleza_comboBox</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>genera_pushButton</tabstop>
  <tabstop>carga_pushButton</tabstop>
  <tabstop>borrarpushButton</tabstop>
  <tabstop>datospushButton</tabstop>
  <tabstop>mayorpushButton</tabstop>
  <tabstop>copiarpushButton</tabstop>
  <tabstop>salirpushButton</tabstop>
 </tabstops>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>salirpushButton</sender>
   <signal>clicked()</signal>
   <receiver>modelo_182</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>859</x>
     <y>596</y>
    </hint>
    <hint type="destinationlabel">
     <x>898</x>
     <y>564</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
